operator/src/main/java/org/bf2/operator/operands/KafkaCluster.java

```diff
@@ -631,6 +632,18 @@ private Map<String, Object> buildKafkaConfig(ManagedKafka managedKafka, Kafka cu
         if (cruiseControlEnabled) {
             config.put("cruise.control.metrics.topic.min.insync.replicas", this.configs.getConfig(managedKafka).cruiseControl.getMetricReporterTopicMinInsyncReplicas());
         }
+
+        // Override broker config from operand override
+        String strimzi = managedKafka.getSpec().getVersions().getStrimzi();
+        Optional.ofNullable(this.overrideManager.getKafkaOverride(strimzi)).map(OperandOverrideManager.Kafka::getBrokerConfig).orElse(Map.of()).forEach((key, value) -> {
```

Comment:
Would the `Optional` be better returned by the `overrideManager`? Or does that really mess with Jackson?

Reply:
Yes I think so, but I think this should be applied to the other getXXXXOverride methods.  I'd prefer this as a separate refactoring in a separate PR.

Reply:
Fine with me